packages/console/src/containers/AppContent/components/Topbar/TenantSelector/CreateTenantModal/index.tsx

```diff
@@ -0,0 +1,30 @@
+import type { TenantInfo } from '@logto/schemas/models';
```

Comment:
nit: Is it ok to put the CreateForm in this Modal element? Looks like the wrapper component does not carry any logic.

Reply:
Merged CreateForm to the CreateTenantModal component to reduce complexity.